src/librustc/dep_graph/graph.rs

```diff
@@ -812,7 +860,7 @@ impl CurrentDepGraph {
                     reads.push(source);
                 }
             }
-            Some(&mut OpenTask::Ignore) | None => {
+            Some(&mut OpenTask::Ignore) | Some(&mut OpenTask::Untracked { .. }) | None => {
```

Comment:
Since this line is getting a little long, it would be more readable to have each arm on its own line.